src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Bit.Core.Settings;
+using LaunchDarkly.Sdk.Server;
+
+namespace Bit.Core.Services;
+
+public class LaunchDarklyFeatureService : IFeatureService, IDisposable
+{
+    private readonly LdClient _client;
+
+    public LaunchDarklyFeatureService(
+        GlobalSettings globalSettings)
+    {
+        var ldConfig = Configuration.Builder(globalSettings.LaunchDarkly?.SdkKey)
+                .Offline(globalSettings.SelfHosted)
```

Comment:
❓ What else should drive offline support here?

Reply:
Not really sure how LaunchDarkly would handle a null SdkKey input, but probably offline if the SdkKey is null or whitespace, too.